code/modules/projectiles/guns/energy/netgun_yw.dm

```diff
@@ -0,0 +1,36 @@
+
+/obj/item/weapon/gun/energy/hunter
+	name = "Hybrid 'Hunter' net gun"
+	desc = "A Hephaestus-designed hybrid of a taser and the energy net gun, usually dubbed 'Hunter' stunner and energy net launcher, \
+			for when you want criminals to stop acting like they're on a 20th century British comedy sketch show."
+	catalogue_data = list(/datum/category_item/catalogue/information/organization/hephaestus)
+	icon = 'icons/obj/gun_vr.dmi'
+	icon_state = "hunter"
+	item_state = "gun" // Placeholder
+	mode_name = "stun"
+
+	fire_sound = 'sound/weapons/eluger.ogg'
+	origin_tech = list(TECH_COMBAT = 4, TECH_MATERIAL = 6, TECH_MAGNET = 4)
+	projectile_type = /obj/item/projectile/beam/stun/blue
+	charge_cost = 240
+	fire_delay = 5
```

Comment:
```suggestion
```
Is not needed as its set in the firemodes list (yes i tested this) and you have set what mode to use from the start mode_name = "stun so this can go poof